### ISSUE TITLE
S3 Path specification required if using athena-express on EKS deployment

### ISSUE THREAD

**Issue by roych98** [ghdna/athena-express#91]
Hey, so long story short, I have a nodejs application which is deployed on an EKS cluster. All nodes have security policies to be able to access S3 directly without having to have an accessKeyId specified within the AWS Config.

I noticed that if I try to construct an AthenaExpress object without specifying an S3 path, an error is thrown as follows:
`throw new TypeError(
      "AWS object not present or incorrect in the constructor"
    );`
From `./lib/helpers.js `within the athena-express project.

If I do specify and S3 path, everything works flawlessly. So I figured that the problem lies within `init.aws.config.credentials.accessKeyId` which is most undefined (Because I don't even need to declare an AWS Config anywhere in the code because my code is running within AWS infrastructure and I allow the communication between my EKS nodes and S3), and thus the error.

Willing to share more information if needed. Thanks.

**Comment by ghdna** [comment]
Right... it assumes accesskeyId is present. I can remove that piece.

**Comment by roych98** [comment]
Awesome, thanks. Otherwise the tradeoff would be that if you use an EKS deployment you just have to specify an S3 path.
